tests/unit_tests/testIndex.js

```diff
@@ -175,16 +175,9 @@ describe('index tests', () => {
 
     const index = require('../../lib/index');
     await index.run();
-
-    /**
-     * We've updated this onboarding tool. ${colors.red(currentVersion)} -> ${colors.green(
-        latestVersion
-      )}. To get the latest experience, run: ${colors.green('npm init nightwatch@latest')
-     */
-
     const output = consoleOutput.toString();
     assert.strictEqual(
-      output.includes('We\'ve updated this obboarding tool. 1.1.2 -> 1.0.2. To get the latest experience, run: npm init nightwatch@latest'),
+      output.includes('We\'ve updated this onboarding tool. 1.1.2 -> 1.0.2. To get the latest experience, run: npm init nightwatch@latest'),
```

Comment:
We should use `${VERSION}` instead of `1.1.2` otherwise this test would fail as soon as we release a new version.